draft-birkholz-rats-corim.md

```diff
@@ -789,10 +791,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+
+This document defines a new registry titled "CoMID Role Types".
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-32767           | Standards Action
+| 32768-4294967295  | Specification Required
+{: #tbl-iana-comid-role-types-reg-procedures title="CoMID Role Types Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Role Types" registry are provided below.
+Assignments consist of an integer value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.tag-creator | RFC-AAAA
+| 1 | comid.creator | RFC-AAAA
+| 2 | comid.maintainer | RFC-AAAA
+| 3-4294967295 | Unassigned
+{: #tbl-iana-comid-role-types-items title="CoMID Role Types Initial Registrations"}
+
+## rim+cbor Media Type Registration
+
+IANA is requested to add the following to the IANA "Media Types" registry {{!IANA.media-types}}.
+
+Type name: application
+
+Subtype name: rim+cbor
+
+Required parameters: none
+
+Optional parameters: none
+
+Encoding considerations: Must be encoded as using {{RFC8949}}. See
+RFC-AAAA for details.
+
+Security considerations: See {{sec-sec}} of RFC-AAAA.
+
+Interoperability considerations: Applications MAY ignore any key
+value pairs that they do not understand. This allows
+backwards compatible extensions to this specification.
+
+Published specification: RFC-AAAA
+
+Applications that use this media type: The type is used by remote attestation procedures, supply chain integrity management systems, vulnerability assessment systems, and in applications that rely on trustworthy endorsements and reference values describing the intended operational state of a system.
+
+Fragment identifier considerations: Fragment identification for
+application/rim+cbor is supported by using fragment identifiers as
+specified by {{Section 9.5 of RFC8949}}.
+
+Additional information:
+
+Magic number(s): first five bytes in hex: 43 4f 52 49 4d
+
+File extension(s): corim
+
+Macintosh file type code(s): none
+
+Macintosh Universal Type Identifier code: org.ietf.corim
+conforms to public.data
+
+Person & email address to contact for further information:
+Henk Birkholz \<henk.birkholz@sit.fraunhofer.de>
+
+Intended usage: COMMON
+
+Restrictions on usage: None
+
+Author: Henk Birkholz \<henk.birkholz@sit.fraunhofer.de>
+
+Change controller: IESG
+
+## CoAP Content-Format Registration
+
+IANA is requested to assign a CoAP Content-Format ID for the CoRIM
+media type in the "CoAP Content-Formats" sub-registry, from the "IETF
+Review or IESG Approval" space (256..999), within the "CoRE
+Parameters" registry {{RFC7252}} {{!IANA.core-parameters}}:
+
+| Media type            | Encoding | ID    | Reference |
+| application/rim+cbor  | -        | TBD1  | RFC-AAAA  |
```

Comment:
Word rim is used for Reference Integrity Manifest.
Is it possible to use application/corim+cbor ?

Reply:
but the Co is redundant to CBOR, right?

CoSWID uses xml+cbor, that seems symmetric